### DISCUSSION TITLE
Discrepancy between documentation and behaviour of `yarn dedupe`

### DISCUSSION THREAD

**Discussion by ShuttleSantalex** [yarnpkg/berry#3007] [Q&A] [answered]
- [ ] I'd be willing to implement a fix

**Describe the bug**

The documentation states the following: `The dedupe command isn’t necessary. yarn install will already dedupe.` see: https://classic.yarnpkg.com/en/docs/cli/dedupe/

But when I run `$ yarn install` on my project, then run `$yarn dedupe`, the `yarn dedupe` command removes a whole bunch of packages that `yarn install` did not. Therefore I am confused.

**To Reproduce**

Install a package that depends on another, then install that same package seperatly, `$ yarn install`, check terminal. then `$yarn dedupe` check terminal. Results of both command should be different while documentation states the contrary.

**Environment if relevant (please complete the following information):**

 - OS: macOS Mojave
 - Node version 12.6.1
 - Yarn version 2.4.1

**Additional context**

**Comment by merceyz** [comment, marked as answer]
That's the docs for Yarn 1.x, the docs for Yarn 2+ is here https://yarnpkg.com/cli/dedupe

**Reply by ShuttleSantalex** [reply]
@merceyz I feel stupid, thanks :)
